docker/lr-pb/Makefile

```diff
@@ -1,17 +1,17 @@
 IMAGE_NAME = lr-pb
-VERSION = 0.1.37
+VERSION = 0.1.38
```

Comment:
We are already having a fragmentation problem for this docker.
https://github.com/broadinstitute/long-read-pipelines/search?l=wdl&q=lr-pb